examples/avr/.cargo/config.toml

```diff
@@ -0,0 +1,8 @@
+[build]
+target = "avr-unknown-gnu-atmega328"
```

Comment:
Should the example directory name more closely match the rustc target (avr -> atmega328)?

Currently nightly only supports this target, but avr-hal maintains some [additional spec files](https://github.com/Rahix/avr-hal/tree/main/avr-specs). I'm unsure if these will all end up being their own rustc platforms, if they can drop the gnu dependency, or if they can be consolidated under a more generic name.

 I'm assuming any AVR support here would only use rustc supported targets, otherwise this repo would likely need to duplicate the spec files.